Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10-slim@sha256:2bac43769ace90ebd3ad83e5392295e25dfc58e58543d3ab326c3330b505283d
```

Comment:
I just want to understand the reasoning why specify the whole docker sha? That would stop it from updating if a security patch comes out for the image...

Reply:
Yeah that's probably just matter of opinion, pinning the image with a hash allows for a reproducible build.

Reply:
By all means it is ^.^